### ISSUE TITLE
[ED][Roles Definition]

### ISSUE THREAD

**Issue by Sylduch-Conseil** [w3c/wai-roles-responsbilities#11]
May be a role can be missing ! 
The person who is in charge of checking that accessibility is taken into account from the beginning of the project till its end. Someone whom yo can ask about accessibility issues: how to follow SC X, are we sure that such and such people will access such and such functionality? 
Someone who is doning outreach through all roles: explaining how important accessibility is, what to do? How to decide that such or such thing should be implemented so that a part of the product is accessible and can be used be people with disabilities.
Another thing could be added: in the roles one could add a trained person in accessibility matters who has a disability herself.

**Comment by bakkenb** [comment]
Ultimately, I would think this should be included in the role of the Product Manager. The problem is that they often don't understand or have experience with PWD or accessibility guidelines in general. But, I guess that is the point of this resource... so that a Product Manager can read about their role and see what their responsibilities are. 

Maybe Sylvie is trying to dig deeper and determine if there should be a role for a person that knows all of the guidelines and best practices on a team so that they can see it all through from a very global level.  Should the role of "Accessibility Expert" or "Accessibility Specialist" be added?

**Comment by dboudreau** [comment]
What we're tying to map out with this resource is what a typical digital production chain might look like, so teams out there can mimic it, and get better at integrating accessibility at every stage of their process... I've yet to see any web development lifecycle that includes someone who's sole responsibility is to overlook how, if or when accessibility needs to be accounted for at every step of a project. The need to add such a role feels unjustified and inadequate. Also, the whole point of these roles is that any team who follows this matrix would actually NOT need such a person on the team, because each team member would know what THEY have o do to make things work.

I agree with Brent that a PM should oversee some of these things, and our matrix will reflect that. I don't think there's room for an accessibility expert or specialist in this resource, but I'm open to arguments as to why anyone might feel otherwise.

**Comment by bakkenb** [comment]
I see and understand your points Denis. I can support that from my perspective, and am okay with not adding such a role.

**Comment by dboudreau** [comment]
Sounds good. Then I'll use my newly obtained powers and close this one. :)

I think the idea of having documentation about what an accessibility champion role might bring to a team is great, but just not within this particular resource.
